model_analyzer/config/generate/model_variant_name_manager.py

```diff
@@ -21,95 +22,79 @@ class ModelVariantNameManager:
 
     def __init__(self):
 
-        # Stored as _variant_names[base_model_name][param_combo] = variant_name_string
-        self._variant_names = {}
+        # Dict of {model_config_name: model_config_dict}
+        self._model_config_dicts = {}
 
-        # Stored as _model_name_index[base_model_name] = current_count_integer
+        # Dict of {base_model_name: current_count_integer}
         self._model_name_index = {}
 
-    def get_model_variant_name(self, base_model_name, param_combo):
+    @classmethod
+    def _from_dict(
```

Comment:
If it is a classmethod, should we remove the underscore?

Reply:
Yes. I've removed it.